CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.8.3] - 2020-02-05
+- Support for construct 2.10 (followed master branch 2dcf92beb727542a9153da923735c1614653b177)
```

Comment:
I am not a huge fan of having the commit hash on the changelog, since it is intended for end-user who might know nothing about git.

Also, this should be tagged as `### Added`.

Finally, I wouldn't list [1.8.3] yet. I would list it under "Unreleased". When the PR is merged, I will pack the release 1.8.3 (but this may not happen on 2020-02-05).

Reply:
Ok.
(I am helped your activity about satellite and am inspired. Thanks. )